test/basic/vpc_create/vpc_create_test.go

```diff
@@ -13,8 +14,18 @@ import (
 
 	cac "github.com/aptible/cloud-api-clients/clients/go"
 	"github.com/aptible/terraform-provider-aptible-iaas/internal/client"
+	"github.com/aptible/terraform-provider-aptible-iaas/test/utils"
 )
 
+func cleanupAndAssert(c context.Context, t *testing.T, terraformOptions *terraform.Options, environmentId, assetId string) {
```

Comment:
nitpick, might wanna keep using the same name `ctx` everywhere, i think we use the var `c` for client in some places